src/components/Staking/StakingStatsBox.tsx

```diff
@@ -111,6 +108,7 @@ const StakingStatsBox: React.FC<IProps> = () => {
   const [totalValidators, setTotalValidators] = useState<string | null>(ZERO)
   const [currentApr, setCurrentApr] = useState<string | null>(ZERO)
 
+  // TODO: Confirm data fetch approach/frequency
```

Comment:
We agreed that this data should be fetched at build time, inside `getStaticProps`.

Reply:
the TODO worked! =) Got it... I see we noted this data should be revalidated daily, will implement for this PR